src/clj/rems/db/licenses.clj

```diff
@@ -112,3 +112,29 @@
 (defn remove-license-attachment!
   [attachment-id]
   (db/remove-license-attachment! {:id attachment-id}))
+
+(defn- get-license-usage [id]
+  ;; these could be db joins
+  (let [resources (->> (db/get-resources-for-license {:id id})
+                       (map :resid)
+                       (map #(db/get-resource {:id %}))
```

Comment:
Worst case, this will make as many DB queries as there are resources. I'm okay with it, since the update license command should be rarely used.

Reply:
yeah that was kinda my thinking as well